### ISSUE TITLE
Double clicking table borders to expand to maximum size for col

### ISSUE THREAD

**Issue by AndrewMohawk** [redcanaryco/mac-monitor#25]
**At a high level -- can you summarize your request?**
I am used to double clicking right hand border of a col and having it auto expand to the size of the largest text in that col and I think this would be a nice feature here as well. See Google Sheets / Excel / etc

**Comment by Brandon7CC** [comment]
Super cool suggestion! Let me look into it and get back to you!

**Comment by Brandon7CC** [comment]
Hey @AndrewMohawk! This looks like it's currently something I'll be limited by with SwiftUI (similar to table column visibility in Ventura). Hopefully next year we'll get an update!
